virtcontainers/qemu.go

```diff
@@ -1090,6 +1095,13 @@ func (q *qemu) hotplugAddCPUs(amount uint32) (uint32, error) {
 		socketID := fmt.Sprintf("%d", hc.Properties.Socket)
 		coreID := fmt.Sprintf("%d", hc.Properties.Core)
 		threadID := fmt.Sprintf("%d", hc.Properties.Thread)
+
+		// If CPU type pseries, we do not set socketID and threadID
```

Comment:
Nit: I'd say "IBM pSeries" as wikipedia tells me there are atleast two other hardware platforms with this name so it doesn't hurt to be specific ;)

Reply:
@jodh-intel : Fixed.